Userland/Libraries/LibWeb/SVG/AttributeParser.cpp

```diff
@@ -122,12 +122,19 @@ void AttributeParser::parse_drawto()
     }
 }
 
+// https://www.w3.org/TR/SVG2/paths.html#PathDataMovetoCommands
 void AttributeParser::parse_moveto()
 {
     bool absolute = consume() == 'M';
     parse_whitespace();
-    for (auto& pair : parse_coordinate_pair_sequence())
-        m_instructions.append({ PathInstructionType::Move, absolute, pair });
+    auto coordinate_pairs = parse_coordinate_pair_sequence();
+    for (size_t i = 0; i < coordinate_pairs.size(); i++) {
+        if (i == 0) {
+            m_instructions.append({ PathInstructionType::Move, absolute, coordinate_pairs[i] });
```

Comment:
specification says that first move in always absolute if you provide path (i.e. multiple coordinates); it seems to be relative for first coordinate in `m` only when itʼs the only coordinate. (—but i might have misread that! 🙇🏻‍♂️)

Reply:
> Start a new sub-path at the given (x,y) coordinates. M (uppercase) indicates that absolute coordinates will follow; m (lowercase) indicates that relative coordinates will follow. If a moveto is followed by multiple pairs of coordinates, the subsequent pairs are treated as implicit lineto commands. Hence, implicit lineto commands will be relative if the moveto is relative, and absolute if the moveto is absolute. If a relative moveto (m) appears as the first element of the path, then it is treated as a pair of absolute coordinates. In this case, subsequent pairs of coordinates are treated as relative even though the initial moveto is interpreted as an absolute moveto.

> Hence, implicit lineto commands will be relative if the moveto is relative, and absolute if the moveto is absolute.

I read this as `M` -> `L` & `m`-> `l`. 
So I think we can just pass the absolute value like this. 

I believe the spec is saying, the very first move of a path is absolute, even if we're moving with `m` but this is already done implicitly because we're starting at 0,0, which makes the first relative move the equivalent of an absolute move. 
Does that make sense @konradekk ?